### ISSUE TITLE
Test failure 106 - Nerve_GIC_example_CoordGIC

### ISSUE THREAD

**Issue by nathan-hello** [GUDHI/gudhi-devel#1128]
The output from `ctest --output-on-failure` is in the attached file.
[LastTest.log](https://github.com/user-attachments/files/16900444/LastTest.log)

The following tests FAILED:
        106 - Nerve_GIC_example_CoordGIC (Subprocess aborted)

If this is known, there happened to be no issues on github. If there is a better place to report this, please tell me. I wished to package this and if this is expected behaviour then I can close the issue.

Many thanks.

**Comment by VincentRouvreau** [comment]
Thanks for the bug report, this issue was not known.
From what I can see in the logs, you do not have CGAL installed (it could be seen also in the CMake output).
The module Nerve_GIC detects if CGAL is installed with the c++ test `#if __has_include(<CGAL/version.h>)`.

It means that you do not have CGAL installed (or not detected by CMake), but that there is a file `CGAL/version.h` somewhere in your system include files.

What is your CMake output ?

**Comment by nathan-hello** [comment]
I apologize but I recompiled and now no longer have the issue. 
I shouldn't have CGAL installed however as `ldconfig -p | grep CGAL` has no output.
It could be perhaps due to the system in place that allows compilation without CGAL. Which is mentioned as a note on the C++ installation page.

**Comment by mglisse** [comment]
> but that there is a file `CGAL/version.h` somewhere in your system include files.

I don't think so, the logic_error only happens if the compiler found neither CGAL/version.h nor hera/bottleneck.h.
Not finding hera should be impossible, it is a submodule (so a good `git clone --recurse-submodules` should download it), and it is bundled with gudhi in gudhi.3.10.1.tar.gz. However, there is a trap: github automatically creates gudhi-release-3.10.1.tar.gz (and a similar zip) that does not contain the sources of the submodules. Maybe we could add a warning to that page (as suggested in https://github.com/orgs/community/discussions/6003#discussioncomment-8818411)?
